app/controllers/users_controller.rb

```diff
@@ -1,4 +1,30 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  def show
+    @user = User.find_by(id: params[:id])
+    return if @user
+
+    flash[:warning] = "Not found user!"
+    redirect_to(root_path)
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to("/users/#{@user.id}")
```

Comment:
```suggestion
      redirect_to user_path(id: @user.id)
```
hoặc gọn nhất là `redirect_to @user` là đủ